capellambse/model/common/accessors.py

```diff
@@ -23,6 +23,7 @@
 
 import abc
 import collections.abc as cabc
+import copy
```

Comment:
I'm not sure if copying the deleted (lost) objects is really needed. If so can we get away without deep-copying?